app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb

```diff
@@ -521,7 +521,7 @@ def hardware_disks(persister_hardware, disks)
     end
   end
 
-  def vm_hardware_guest_devices(persister_hardware, vm, addresses, host)
+  def vm_hardware_guest_devices(persister_hardware, vm, addresses, _host)
```

Comment:
This is an internal method so we can just drop the `host` argument if it isn't used

Reply:
I'm always paranoid about interface compliance, but ok.

Reply:
```
../manageiq-providers-ovirt/app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb:    vm_hardware_guest_devices(persister_hardware, vm, addresses, host) unless template
../manageiq-providers-ovirt/app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb:  def vm_hardware_guest_devices(persister_hardware, vm, addresses, host)
```